Makefile

```diff
@@ -2,6 +2,7 @@ export GO111MODULE=on
 
 # If VERSION or LDFLAGS change, please also change
 # npm/build.
+GOARCH = "amd64"
```

Comment:
I'd change the name of this (`ARCH`) since it doesn't impact any of the go build steps. This makefile variable is not be exported as an environment variable anywhere, and the one place we set a GOARCH env var is already fixed to amd64 (in create-release-assets).